src/store/actions/posts.js

```diff
@@ -0,0 +1,141 @@
+import axios from 'axios';
+import * as actionTypes from './actionTypes';
+import HttpsProxyAgent from 'https-proxy-agent';
+import { $CombinedState } from 'redux';
+var agent = new HttpsProxyAgent("http://localhost:3000");
+//import firebase from 'firebase'
+export const postStart = () => {
+    return {
+        type: actionTypes.POST_START
+    };
+};
+
+export const postSuccess = (post) => {
+    return {
+        type: actionTypes.ADD_POST,
+        post: post,
+    };
+};
+
+export const fetchPostFailed = (error) => {
+    return {
+        type: actionTypes.FETCH_POST_FAILED,
+        error: error
+    };
+};
+
+export const post = (title, content, token, userId) => {
+    return dispatch => {
+        dispatch(postStart());
+        const data = {
+            title: title,
+            content: content,
+            userId: userId
+        }
+        axios.post('https://assignment-87476.firebaseio.com/posts.json?auth=' + token, data, {
```

Comment:
need to create one helper for API call `axios`

Reply:
Need to work on it.

Reply:
Please you need to cover this point

Reply:
Committed this change now.